test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -390,6 +390,8 @@ class ProtoConverter
 	/// Contains typed parameter list to be passed to callee functions
 	std::ostringstream m_typedParamsExternal;
 	std::ostringstream m_typedParamsPublic;
+	/// Contains parameter list to be passed to callee functions
+	std::ostringstream m_paramsExternal;
```

Comment:
Just for clarify, wouldn't it be better to explicitly call this "untyped"? I.e. currently it sounds like a more general version of `m_typedParamsExternal` but I guess it's not?
```suggestion
	std::ostringstream m_untypedParamsExternal;
```

Or maybe even more specific to suggest that these are parameters in a format suitable to pass into a call, though I have not good suggestion at hand.